Add Content (#1)
danielehc/katacoda-scenarios: consul_service_mesh_2/step6.md

#### Configure port forwarding for your frontend service

To access the app running inside your Consul service mesh, you will setup port forwarding.

`export IP_ADDR=$(hostname -I | awk '{print $1}')`{{execute}}

`kubectl port-forward service/web 9090:9090 --address ${IP_ADDR}`{{execute}}

> This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.

Reviewer: ```suggestion
This will forward port `80` from `service/hashicorp-consul-ui` at port `80` to your test machine.
```

Reviewer 2: I am confused. This step is to add port forwarding for port 9090 and access the Dashboard (re-checked on my local copy). So the port described should be 9090. and the service `service/web`

For scenario 2 port forwarding for 80 is made in the provision script, that's why is not mentioned.

Reviewer 2: ```suggestion
This will forward port `9090` from `service/web` at port `9090` to your test machine.

You can now open the [Dashboard](https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com/ui) tab to be redirected to the web application UI.
```

Reviewer: Cool.  My only input was on preposition selection.

> This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.
> 

Reviewer 2: ```suggestion
This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.

You can now open the [Dashboard](https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com/ui) tab to be redirected to the web application UI.
```

This will forward port `9090` of `service/web` on port `9090` of your test machine.
This will forward port `80` of `service/hashicorp-consul-ui` on port `80` of your test machine.

> You can now open the [Dashboard](https://[[HOST_SUBDOMAIN]]-9090-[[KATACODA_HOST]].environments.katacoda.com/ui) tab to be redirected to the web application UI.

Author: I think I accidently deleted this...

Reviewer 2: Re-added above

> You can now open the Consul UI tab to be redirected to the Consul UI.

Reviewer 2: ```suggestion
```
